js/util/theme.js

```diff
@@ -4,7 +4,13 @@ if (typeof require !== 'undefined') {
 
 function shouldEnableDarkMode () {
   var hours = new Date().getHours()
-  return hours > 21 || hours < 6
+  let isAuto = false
+
+  settings.listen('darkMode', function (value){
+    isAuto = (value === undefined || value === true || value >= 0)
+  })
```

Comment:
The setting is already checked in initialize(), which is the only place that calls this function, so it should be fine to remove this. It would also probably make sense to rename this function to something like `isNight()` if you want to do that.